tests/test_surface.py

```diff
@@ -0,0 +1,68 @@
+import os
+
+import oddt
+from oddt.surface import (generate_surface_marching_cubes,
+                          find_surface_residues)
+from sklearn.utils.testing import (assert_array_equal,
+                                   assert_array_almost_equal,
+                                   assert_equal,
+                                   assert_almost_equal,
+                                   assert_in)
+import pandas as pd
+import numpy as np
+
+
+test_data_dir = os.path.dirname(os.path.abspath(__file__))
+protein = next(oddt.toolkit.readfile('pdb', os.path.join(
+    test_data_dir, 'data/pdbbind/10gs_protein.pdb')))
+protein.protein = True
+protein.addh(only_polar=True)
+
+def test_generate_surface_marching_cubes():
+    """Tests generating surfaces"""
+    verts1, faces1 = generate_surface_marching_cubes(protein, scaling=1., probe_radius=1.4)
```

Comment:
It's a good practice to specify all arguments (the ones with default values as well). This way you have bigger control over your tests. If we decide to change default value, the function still works but your test will fail.

Also you should provide some tests for all arguments:
- [ ] `remove_hoh` - add at least one test case with water removal. You can e.g. check verts and faces shape, like you do for `scaling`. I think it is also a good idea to check what happens if we use molecule without water residues and `remove_hoh=True` (expected behavior: water removal shouldn't change the surface)
- [ ] `probe_radius` - test different radii, especially 0.